### PR TITLE
Refs [Task] [44182] Build task/register

### PR DESCRIPTION
**Pull request by tuanna-1205**
https://edu-redmine.sun-asterisk.vn/issues/44182

### CODE REVIEW

**Comment by tuanna-1205** [comment]
ready

File: app/views/blocks/register.pug [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    .register__heading
+      .register__heading__inner
+        +formHeading({class1: 'register__heading-title', title: 'Đăng ký', src: '../../images/titleleft-dark.png'})
+      a.btn.btn-active.register__heading-btn(href="#")  Đăng nhập
+    form.form(action="#")
+      h3.form-title Thông tin cá nhân
+      +formGroup({label: 'Tên trước*'})
+      +formGroup({label: 'Tên sau*'})
+      +formGroup({label: 'Email*'})
+      +formGroup({type: 'password', label: 'Password*'})
```
Comment (vietnq-1097) [inline app/views/blocks/register.pug]: Bổ sung thêm id cho form group nhé
https://stackoverflow.com/questions/36334658/label-for-and-input-id

File: app/views/index.pug [hunk 2]
```diff
@@ -11,4 +11,5 @@ block container
 	include ./blocks/list-products2
 	include ./blocks/blog-posts
 	include ./blocks/sponsor-section
-	include ./blocks/log-in
\ No newline at end of file
+	include ./blocks/log-in
+	include ./blocks/register
```
Comment (vietnq-1097) [inline app/views/index.pug]: endline

File: app/views/blocks/register.pug (hunk 1 above)
Comment (tuanna-1205) [reply app/views/blocks/register.pug]: oke ạ

**Comment by tuanna-1205** [comment]
ready

**Pull request merged**